Documentation/ExtensionConfiguration/Index.rst

```diff
@@ -62,15 +62,14 @@ Extension icon
 ==============
 
 Not as important as the extension declaration file above, but every extension can
-feature an icon using a PNG file. This image should be located in the root
-directory of the extension as well, and must be named :file:`ext_icon.png`.
-Choose or create an image of 64px width by 64px height.
+feature an icon using a svg file. This image should be located at
+:file:`Resources/Public/Icons/Extension.svg`.
 
 .. note::
 
-   Newer versions of TYPO3 support alternative formats and file locations.
-   However, for the sake of simplicity we will stick to the specification outlined
-   above.
+   PNG and GIF files are also allowed as Extensions icon. They are stored in
```

Comment:
```suggestion
   PNG and GIF files are also allowed as extension icon. They should be stored in
```

Reply:
Shouldn't that be "[if used...], they have to be stored in"? "Should" suggests that I could store them somewhere else, which isn't the case as far as I know,

Reply:
Indeed, extension icons *must* be in `Public`, otherwise the simply won't work.

Reply:
If you use `helhum/typo3-secure-web` that is true. But for the time being they also work if they are in the original location. However I prefere to have them in Public, too....